requirements/dev.txt

```diff
@@ -20,3 +20,6 @@ livereload==2.3.2
 
 # Avoid eating cpu with live reloading
 watchdog==0.8.3
+
+# QA Helper
```

Comment:
Any reason the same version in the root `requirements.txt` does not work? https://github.com/brianjgeiger/osf.io/blob/0.67.0/fix-QA-info-logging/requirements.txt#L51